docs/compose/StarRating/README.md

```diff
@@ -69,3 +69,20 @@ BpkHotelRating(
     size = BpkStarRatingSize.Small,
 )
 ```
+
+Example of a `BpkInteractiveStarRating`:
+
+```Kotlin
+import net.skyscanner.backpack.compose.starrating.BpkInteractiveStarRating
+import net.skyscanner.backpack.compose.starrating.BpkStarRatingSize
+
+
+BpkInteractiveStarRating(
+    onRatingSelected = { selectedRating = it },
```

Comment:
can you either include the definition of the `selectedRating` property here or make this an empty lambda?

Reply:
Ah sorry! Btw: `selectedRating` was renamed as `rating` for consistency, and `rating` is already here as an attribute. So do I still need to define it then?

Reply:
Discussed this with Vitaliy, fixing now!